addons/l10n_ec_ats/tablas_model/tablas.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields, api
+
+#class res_partner(models.Model):
+#    _inherit = 'res.partner'
+#    x_tipoid = fields.Char('Tipo Identificación', help="Ingrese el tipo de identificación", required=True)

```

Comment:
this should be on your models directory and in only one file, why do you have all your code in separated files?, there is any reason?...